yogstation/code/modules/guardian/abilities/special/pocket.dm

```diff
@@ -0,0 +1,387 @@
+GLOBAL_VAR_INIT(pocket_dim, 1)
+GLOBAL_LIST_EMPTY(pocket_mirrors)
+
+/proc/get_final_z(atom/A)
+	var/turf/T = get_turf(A)
+	if(T)
+		return T.z
+	return A.z
+
+/area/hippie/pocket_dimension
```

Comment:
wow great port keeping the word "hippie" lul

Reply:
ok fine

Reply:
ez done